### ISSUE TITLE
Create Search Query for course redirection

### ISSUE THREAD

**Issue by a18osken** [HGustavs/LenaSYS#9913]
When a user attempts to access a course by using a shortened URL such as `../sh/?c=databaskonstruktion` the c element should be compared to the database's list of redirection keywords, We want to look for any unique match and if it exists we should redirect to that course, likewise if there is a unique match to the keyword list that is not stored we should store that match. So that if ,for example,  `../sh/?c=databask` is entered as a URL, databask gets added to the list of the course's redirection keywords. 

If we get more than one match to the URL we should display an error.

**Comment by a18osken** [comment]
I realized that we also need to have the list of keywords created before we can compare and populate it. We could maybe do this in this issue because of the pain of synchronizing databases across the board, or I'll create a separate Issue for making and populating the table with its first keywoards, e.g : databaskonstruktion & dbk.

**Comment by HGustavs** [comment]
Conditions to test for:

* We must be very careful so that a previously working redirection does not break when we at a later point in time add a new course with a partially matching name
* We must make sure that redirections that do not match anything will throw an error message and not redirect to non-existent pages

**Comment by a19simma** [comment]
Is a disabled course when visibility is set to 0?
